gymnasium/core.py

```diff
@@ -194,14 +195,18 @@ def __enter__(self):
         """Support with-statement for the environment."""
         return self
 
-    def __exit__(self, *args):
+    def __exit__(self, *args: List[Any]):
```

Comment:
https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values

so it should be 
`def __exit__(self, *args: Any):`

(I'm not sure if Any wouldn't be automatically filled in any type checker, but doesn't hurt to include it)

Reply:
Wow impress check, will update